src/docs/08-operation-guide/01-setup.md

```diff
@@ -0,0 +1,348 @@
+---
+layout: default
+title: Worker service
+permalink: /docs/operation-guide/setup
+---
+
+# Cluster Setup
+
+This section will help to understand what you need for setting up a Cadence cluster.
+
+You need to understand some key config options in Cadence server. There are two main types of configs in Cadence server, static config and dynamic config.
+
+Also, you need understand Cadence’s dependency --- a database(Cassandra or SQL based like MySQL/Postgres) and a metric server(typically Prometheus). Cadence also needs ElastiCache+Kafka if you need [Advanced visibility feature to search workflows](/docs/concepts/search-workflows/). And Cadence also depends on a blob store like S3 if you need to enable [archival feature](docs/concepts/archival/).
+
+## Cadence server configuration
```

Comment:
Because this is a long document, I would suggest dropping this header and upgrading each of the subheaders below to `##` as these will have quick links in the leftside navigation.

Reply:
Good idea, done!